models/QuizModel.php

```diff
@@ -754,4 +754,36 @@ public function get_first_or_last_attempt( int $quiz_id, int $user_id = 0, strin
 		);
 		return $attempt;
 	}
+
+	/**
+	 * Get total number of quizzes by course id
+	 *
+	 * @since 2.2.0
+	 *
+	 * @param int|array $course_id Course id or array of course ids.
+	 *
+	 * @return int
+	 */
+	public static function get_quiz_count_by_course( $course_id ) {
+		global $wpdb;
+
+		$and_clause = is_array( $course_id ) ? ' AND post_parent IN (' . implode( ',', $course_id ) . ')' : "AND post_parent = $course_id";
```

Comment:
`prepare_in_clause` has a default sanitization check.
We can use `QueryHelper::prepare_in_clause( $course_id )` instead of `implode( ',', $course_id )`